tools/rsyslogd.c

```diff
@@ -1524,13 +1525,13 @@ initAll(int argc, char **argv)
 			fprintf (stderr, "rsyslogd: the -w command line option will go away "
 				 "soon.\nPlease use the global(net.permitWarning=\"off\") "
 				 "configuration parameter instead.\n");
-			glbl.SetOption_DisallowWarning(0);
+			// glbl.SetOption_DisallowWarning(0);
```

Comment:
as mentioned in previous PR-wide comment, the commented out code should go away. Error message should be reworded to "Option *has* gone away".